pkg/smartcontract/manifest/standard/nep11.go

```diff
@@ -0,0 +1,152 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+var nep11Base = &Standard{
+	Manifest: manifest.Manifest{
+		ABI: manifest.ABI{
+			Methods: []manifest.Method{
+				{
+					Name: "balanceOf",
+					Parameters: []manifest.Parameter{
+						{Type: smartcontract.Hash160Type},
+					},
+					ReturnType: smartcontract.IntegerType,
+					Safe:       true,
+				},
+				{
+					Name:       "decimals",
+					ReturnType: smartcontract.IntegerType,
+					Safe:       true,
+				},
+				{
+					Name:       "properties",
+					ReturnType: smartcontract.MapType,
+					Parameters: []manifest.Parameter{
+						{Type: smartcontract.ByteArrayType},
+					},
+					Safe: true,
+				},
+				{
+					Name:       "symbol",
+					ReturnType: smartcontract.StringType,
+					Safe:       true,
+				},
+				{
+					Name:       "tokensOf",
+					ReturnType: smartcontract.AnyType, // Iterator
+					Parameters: []manifest.Parameter{
+						{Type: smartcontract.Hash160Type},
+					},
+					Safe: true,
+				},
+				{
+					Name:       "totalSupply",
+					ReturnType: smartcontract.IntegerType,
+					Safe:       true,
+				},
+				{
+					Name: "transfer",
+					Parameters: []manifest.Parameter{
+						{Type: smartcontract.Hash160Type},
+						{Type: smartcontract.ByteArrayType},
+					},
+					ReturnType: smartcontract.BoolType,
+				},
+			},
+			Events: []manifest.Event{
+				{
+					Name: "Transfer",
+					Parameters: []manifest.Parameter{
+						{Type: smartcontract.Hash160Type},
+						{Type: smartcontract.Hash160Type},
+						{Type: smartcontract.IntegerType},
+						{Type: smartcontract.ByteArrayType},
+					},
+				},
+			},
+		},
+	},
+	Optional: []manifest.Method{
+		{
+			Name: "properties",
+			Parameters: []manifest.Parameter{
+				{Type: smartcontract.ByteArrayType},
+			},
+			ReturnType: smartcontract.MapType,
+			Safe:       true,
+		},
+		{
+			Name:       "tokens",
+			ReturnType: smartcontract.AnyType,
+			Safe:       true,
+		},
+		{
+			Name: manifest.MethodOnNEP11Payment,
```

Comment:
Not sure this method belong here.

Reply:
It's a receiver method, so it's not for NEP11. But if our NEP11 token wants to implement his method (and to become a receiver), it would be good to check its signature, so I think we may leave it here(as optional method).